Makefile.codegen

```diff
@@ -37,11 +37,15 @@ go-generate:
 generate-client: codegen-clientset codegen-config fmt importfmt ## Generate the client
 
 codegen-clientset: build-codegen ## Generate the k8s types and clients
+	# sleep for one second to prevent collision with previous codegen's replacement of go.mod
+	sleep 1
```

Comment:
Is that really needed?

Reply:
Sometimes, yes - the go.mod juggling seems to not always be completed by the time the next target starts running.

Reply:
I find this odd. These are really independent processes and executions of the codegen binary. A new target is not starting before the previous has completed. How would the files not be updated by then?